lib/src/main/java/vmath/Mat3.java

```diff
@@ -1,5 +1,7 @@
 package vmath;
 
+import org.jetbrains.annotations.NotNull;
```

Comment:
This would make the code dependent on JetBrains's annotations. Could you remove this?